src/components/routers/WebhookRouter.tsx

```diff
@@ -305,18 +305,18 @@ export default class WebhookForm extends React.Component<WebhookRouterFormProps,
         const linkText = this.state.method === Methods.GET ? baseText : `${baseText} and body`;
 
         return (
-            <span>
+            <React.Fragment>
```

Comment:
Constants and Fragments can be considered stylistic changes but I'd argue this is important housekeeping since constants improve code readability/allow us to reuse values (esp in tests) and Fragments reduce the amount of HTML output to the page.